src/main/java/xyz/thepathfinder/android/Action.java

```diff
@@ -42,18 +46,19 @@
      * @param services   a pathfinder services object.
      */
     protected Action(JsonObject actionJson, PathfinderServices services) {
-        logger.finest("Constructing action: " + actionJson.toString());
+        logger.info("Constructing action: " + actionJson.toString());
 
         this.status = Action.getStatus(actionJson);
         this.latitude = Action.getLatitude(actionJson);
         this.longitude = Action.getLongitude(actionJson);
```

Comment:
Is there a GMaps or Android location type class we could wrap latitude and longitude into that would be more natural then storing two separate fields? JS has google.maps.LatLng and iOS has CLLocationCoordinate2D